src/protocols/secure_channel/PairingSession.cpp

```diff
@@ -78,6 +81,20 @@ void PairingSession::Finish()
     if (err == CHIP_NO_ERROR)
     {
         VerifyOrDie(mSecureSessionHolder);
+        DeviceLayer::ChipDeviceEvent event;
+        event.Type                                   = DeviceLayer::DeviceEventType::kSecureSessionEstablished;
+        event.SecureSessionEstablished.TransportType = to_underlying(address.GetTransportType());
+        event.SecureSessionEstablished.SecureSessionType =
+            to_underlying(mSecureSessionHolder->AsSecureSession()->GetSecureSessionType());
+        if (mSecureSessionHolder->AsSecureSession()->GetSecureSessionType() == Transport::SecureSession::Type::kCASE)
+        {
+            event.SecureSessionEstablished.PeerNodeId  = mSecureSessionHolder->GetPeer().GetNodeId();
+            event.SecureSessionEstablished.FabricIndex = mSecureSessionHolder->GetPeer().GetFabricIndex();
+        }
```

Comment:
please ensure all fields are initialized. Even if not CASE, we should probably have known values for nodeid/fabriocindex.

Reply:
Done